modules/thunder_updater/thunder_updater.install

```diff
@@ -106,5 +110,12 @@ function thunder_updater_update_8001() {
   // Finally remove Thunder Updater!
   $module_installer->uninstall(['thunder_updater']);
 
+  // @todo This can be removed when
+  //   https://www.drupal.org/project/drupal/issues/3063734 lands.
+  // Remove thunder_updater from the update schema list.
+  register_shutdown_function(function () {
```

Comment:
Needs to move to a new update function because this has occurred already. However - if the module is uninstalled we can't do the update here because the module has been uninstalled. So we need this here - but also maybe an update in the profile that does the clean up.

Reply:
I think we only need it in the profile, because of update dependencies we ensure that the thunder_updater hook runs at the very beginning